content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -440,6 +440,8 @@ link:../../using/using-credentials#configuring-credentials[configured in
 Jenkins] can be handled in Pipelines for immediate use. Read more about using
 credentials in Jenkins on the link:../../using/using-credentials[Using
 credentials] page.
+.The correct way to handle credentials in Jenkins
```

Needs an empty line before the video so that it does not flow into the preceding paragraph
```suggestion

.The correct way to handle credentials in Jenkins
```